### ISSUE TITLE
[Feature Request] Public API

### ISSUE THREAD

**Issue by rarDevelopment** [backloggd-manager/backloggd-issue-tracker#33]
It'd be useful to have a public API to be able to have external access to more events on the site.

**Comment by yaserahmady** [comment]
Since almost 100% of data about games is coming straight from IGDB, what I wish for in a Backloggd API is:

- Getting a game's rating, ratings count, reviews, lists with the game in it and every Backloggd-specific data about the game
- Managing lists, adding games to lists, removing them, reordering them, adding notes, …

**Comment by rarDevelopment** [comment]
> Since almost 100% of data about games is coming straight from IGDB, what I wish for in a Backloggd API is:
> 
> 
> 
> - Getting a game's rating, ratings count, reviews, lists with the game in it and every Backloggd-specific data about the game
> 
> - Managing lists, adding games to lists, removing them, reordering them, adding notes, …

Agreed on all counts. An API definitely has to lean more Backloggd-specific. IGDB can serve up that data separately if a client of this API needs it.

I think it'd be ideal to have a way to pull in our own reviews for sharing with others via automation, among a host of other things.
